wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx

```diff
@@ -22,10 +22,13 @@ export function CallSc(props: PromptRequestData) {
     Description,
   } = props;
 
-  const toAddInHeigthDescription = Description ? 200 : 0;
+  // Convert nano mass to mass
+  const coinsInMass = coinsInNanoMass / 1000000000;
```

Comment:
can you see if it's possible to use the massa-web3 convert function?

Reply:
https://github.com/massalabs/massa-web3/blob/815357a17b0b44be8759a4abcc643291f7e57c4f/src/utils/converters.ts#L50